lib/hyperswitch.js

```diff
@@ -378,9 +378,9 @@ HyperSwitch.prototype._request = function(req, options) {
             reqHandlerPromise = P.try(handler, [childHyperSwitch, childReq]);
         }
 
-        reqHandlerPromise = self._wrapInMetrics(reqHandlerPromise, match, req)
+        reqHandlerPromise = childHyperSwitch._wrapInMetrics(reqHandlerPromise, match, req)
         .then(function(res) {
-            self.log('trace', {
+            childHyperSwitch.log('trace', {
```

Comment:
I made it trace/hyper/response, since technically what we are tracing here is a successful response.